.github/workflows/ansible-lint-sap_swpm.yml

```diff
@@ -24,15 +24,19 @@ jobs:
 
     steps:
       - name: Check out the code
-        uses: actions/checkout@v2
+        uses: actions/checkout@main
```

Comment:
```suggestion
        uses: actions/checkout@v3
```

Reply:
Closing for now, see https://github.com/sap-linuxlab/community.sap_install/pull/419#discussion_r1270607762.